src/emulator/host/include/host/config.h

```diff
@@ -53,4 +53,5 @@ struct Config {
     std::string background_image = {};
     float background_alpha = 0.300f;
     int icon_size = 64;
+    bool discord_rich_presence = false;
```

Comment:
Think this could be optional?

Reply:
Yes moreb set in on tue by default

Reply:
sure. i'm not sure having it on by default is a good idea zang, but if the devs are ok with it then ok

Reply:
have it on by default